test/inductor/test_cpp_wrapper.py

```diff
@@ -3,7 +3,6 @@
 import unittest
 from typing import NamedTuple
 
-import torch._dynamo
```

Comment:
can replace this by import torch since i believe torch is still needed here and not imported anywhere else